src/js/utils.js

```diff
@@ -194,7 +194,7 @@ function isNull(x) {
 }
 
 function isUndefined(x) {
-    return x === undefined
+    return typeof(x) == 'undefined';
```

Comment:
Ok, I see my comment could be misinterpreted which you did. :) 
I liked the previous way better :)